mfr/extensions/image/static/js/jquery.mousewheel.min.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * jQuery Mousewheel 3.1.13
+ *
+ * https://github.com/jquery/jquery-mousewheel/blob/3.1.13/jquery.mousewheel.min.js
```

Comment:
`jquery.mousewheel` is a general (and powerful) library for handle (and customize) mouse wheel events. We defined scroll based zoom by ourself and applied it directly to the zoomed image by `jquery.zoom` library.